MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -1167,69 +1115,49 @@ private void SetAudioInfoFromTags(MediaInfo audio, Dictionary<string, string> ta
 
             audio.People = people.ToArray();
 
-            var albumArtist = FFProbeHelpers.GetDictionaryValue(tags, "albumartist");
-            if (string.IsNullOrWhiteSpace(albumArtist))
-            {
-                albumArtist = FFProbeHelpers.GetDictionaryValue(tags, "album artist");
-            }
-
-            if (string.IsNullOrWhiteSpace(albumArtist))
-            {
-                albumArtist = FFProbeHelpers.GetDictionaryValue(tags, "album_artist");
-            }
-
-            if (string.IsNullOrWhiteSpace(albumArtist))
-            {
-                audio.AlbumArtists = Array.Empty<string>();
-            }
-            else
-            {
-                audio.AlbumArtists = SplitArtists(albumArtist, _nameDelimiters, true)
-                    .DistinctNames()
-                    .ToArray();
-            }
+            // Set album artist
+            var albumArtist = tags.GetFirstNotNullNorWhiteSpaceValue("album_artist", "albumartist", "album artist");
```

Comment:
Is the change in ordering intentional? To match the previous behaviour it should be this
```suggestion
            var albumArtist = tags.GetFirstNotNullNorWhiteSpaceValue("albumartist", "album artist", "album_artist");
```

Reply:
It's not, I'll revert it to the previous behavior.